modular_skyrat/modules/pixel_shift/code/pixel_shift.dm

```diff
@@ -57,3 +71,19 @@
 			if(pixel_x >= -16 + base_pixel_x)
 				pixel_x--
 				is_shifted = TRUE
+
+	// Yes, I know this sets it to true for everything if more than one is matched.
+	// Movement doesn't check diagonals, and instead just checks EAST or WEST, depending on where you are for those.
+	if(pixel_y > 8)
```

Comment:
8 should be a define, here. -8 could just be -`DEFINE_NAME_HERE`, but yeah, it's the kind of thing you'd want as a define.

The 16s you see above would also be good to have as defines.